openwisp_monitoring/device/static/monitoring/css/device-map.css

```
@@ -67,3 +65,356 @@
+  transform: translate(-50%, -50%);
+  width: 90vw;
+  height: 88vh;
+  z-index: 99998;
+  display: flex;
+  flex-direction: column;
+  min-width: 320px;
+  min-height: 375px;
+}
+#floorplan-container {
+  position: relative;
+  background: white;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)